modules/data.remote/inst/download_SMAP_from_gee.R

```diff
@@ -0,0 +1,110 @@
+##'@name download_SMAP_from_gee
```

Comment:
If this function is virtually identical to `download_SMAP_gee2pecan.R` then why do we have both (which will be really confusing to users)? If the old one doesn't work, wouldn't it make more sense to fix it than to create a duplicate? Also, by creating a duplicate I can't see the diff of what's old vs what you changed. Finally, is there any reason this should be in `inst` instead of `R` if its now ready to use?

Reply:
And you should add yourself to the author list

Reply:
`download_SMAP_from_gee` has been re-integrated into `download_SMAP_gee2pecan.R`.  I also added my name to the author list.